src/modules/CreateArticle.js

```diff
@@ -4,14 +4,27 @@ const CreateArticle = {
   async create(e, dispatch) {
     e.preventDefault();
     let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+
+    const toBase64 = (file) =>
```

Comment:
please make this a separate function, so it is not a part of the `create` function. I would suggest moving it to a separate module and import it here

Reply:
Good call, we've done this now